python/tvm/relax/transform/transform.py

```diff
@@ -233,6 +233,29 @@ def ToNonDataflow() -> tvm.ir.transform.Pass:
     return _ffi_api.ToNonDataflow()  # type: ignore
 
 
+def TopologicalSort(order="depth-first", direction="from-inputs") -> tvm.ir.transform.Pass:
+    """Sort relax.Dataflow blocks
```

Comment:
To be more precise, "Sort bindings in relax.Dataflow blocks with the specified order"?

Reply:
Good call, and updated.